scripts/localdev.py

```diff
@@ -970,21 +970,81 @@ def build_scilla(config, driver, tag):
                capture_output = False)
     return image_name
 
+def build_native_scilla(config):
+    build_env = os.environ.copy()
+    build_env.update(config.default_env)
+
+    if not build_env.get("VCPKG_ROOT"):
+        raise GiveUp("Environment variable VCPKG_ROOT must be defined to point to vcpkg")
+
+    build_env["SCILLA_REPO_ROOT"] = SCILLA_DIR
+
+    # Cleanup
+    shutil.rmtree(os.path.join(SCILLA_DIR, "vcpkg_installed"), ignore_errors = True)
+    shutil.rmtree(os.path.join(SCILLA_DIR, "_build"), ignore_errors = True)
```

Comment:
This is going to lead to extremely long build times (esp on OS X) if we don't check if it's been done first - you probably want to introduce a sentinel here, because otherwise every time you invoke build_native_scilla() you will need to wait for deps to be built (and this includes every time you want to test a change to scilla)

Reply:
If vcpkg has already been installed (either from a previous compilation or Zilliqa), it would simply be installed. Anything new or different will be compiled (and be used by vcpkg in subsequent installs).
The scripts/build_deps.sh in scilla does the same actually and checks if it exists or not so this doesn't really make much of a difference.

The reason for deleting both (predominantly _build) is because there have been strange issues related to ocaml/dune cache.

Reply:
I don't think it does - it calls `scripts/vcpkg.sh` which does:

```
# If already installed, exit early.
if [[ -d vcpkg_installed ]]
then
    echo "Found vcpkg_installed, not installing again"
    exit 0
fi
```

but good point - sorry - I'd not twigged that vcpkg will cache separately from vcpkg_installed - in that case, np (though if the problem is in the build cache, I suspect you will simply regenerate it here .. )